README.md

```diff
@@ -1 +1,57 @@
-# bubble-sort
\ No newline at end of file
+# BUBBLE SORT
+
+This is our first ruby project Bubble Sort we built a method #bubble_sort that takes an array and returns a sorted array. It must use the bubble sort methodology. This is to demonstrate the authors basic understanding of the ruby language concepts.
+
+The project has methods that takes an array as arguments and sorts them in an ascending order using the bubble sort Algorithim.
+
+# Built With
+
+> Ruby
+> rubocop.yml
+
+
+# Live Demo
+
+[@tnyandoro](https://repl.it/)
+
+# Getting started
+
+To get a local copy of this repo, please run the following command in your terminal
+
+
+$ cd<folder>
+
+$ git clone [](https://github.com/tnyandoro/bubble-sort/tree/feature/bubble-sort_app)

```

Comment:
The result of this line is `git clone ` without the repository URL.
Change it with 
```
git clone https://github.com/tnyandoro/bubble-sort/tree/feature/bubble-sort_app
```

Complete your README with how to run your code